config_defaults/subtests/docker_cli/stop.ini

```diff
@@ -1,7 +1,7 @@
 [docker_cli/stop]
 docker_timeout = 60
 #: delay before this test starts using created containers
-wait_start = 3
+wait_start = 10
```

Comment:
FWIW, I'm totally against those short timeouts, esp. in CI, nobody's watching/waiting, so go ahead and make them excessively long.